open-media-match/omm_config.py

```diff
@@ -0,0 +1,18 @@
+from threatexchange.signal_type.pdq.signal import PdqSignal
+from threatexchange.signal_type.md5 import VideoMD5Signal
+
+# Database configuration
+PRODUCTION = True
+DBUSER = "media_match"
+DBPASS = "hunter2"
+DBHOST = "db"
+DBNAME = "media_match"
+DATABASE_URI = f"postgresql+psycopg2://{DBUSER}:{DBPASS}@{DBHOST}/{DBNAME}"
```

Comment:
Here we may also want to use env variables instead of set values since this should allow us to inject variables on start-up, but still allow default when working locally.